src/Templating/Helper/BlockHelper.php

```diff
@@ -138,6 +138,9 @@ public function renderEvent(string $name, array $options = []): string
     {
         $eventName = sprintf('sonata.block.event.%s', $name);
 
+        /**
+         * @psalm-var BlockEvent
```

Comment:
Not sure if this is the best way to do it. PHPStan detects correctly the type, but psalm not. So if I do a normal assert it fails phpstan.

Reply:
A PR can be made to add the stub https://github.com/phpstan/phpstan-symfony/blob/1.2.x/stubs/Symfony/Component/EventDispatcher/EventDispatcherInterface.stub
in the psalm-symfony plugin
https://github.com/psalm/psalm-plugin-symfony/tree/master/src/Stubs/common/Component

Do you want to do it or I do ?

Reply:
Can you do it? anyway I think we can have this and maybe remove later? Given it is a problem with a missing Stub, shall we consider this a suppress instead? so when it gets fixed we will get an error to help us removing this code. wdyt?

Reply:
https://github.com/psalm/psalm-plugin-symfony/pull/254 ; psalm-suppress might indeed help us to remove this code later

Reply:
Done, changed to a suprress instead.